CHANGELOG.md

```diff
@@ -2,4 +2,10 @@
 - Migrate to Vidarr
 
```

Comment:
Does CHANGELOG.md go from the oldest to the latest? I guess it's fine but a bit unorthodox

Reply:
I updated the changelog, good catch!